src-vue/src/views/SettingsView.vue

```diff
@@ -36,37 +32,7 @@ export default defineComponent({
     },
     methods: {
         async updateGamePath() {
-            // Open a selection dialog for directories
-            const selected = await open({
-                directory: true,
-                multiple: false,
-                defaultPath: await appDir(),
-            });
-            if (Array.isArray(selected)) {
-                // user selected multiple directories
-                alert("Please only select a single directory");
-            } else if (selected === null) {
-                // user cancelled the selection
-            } else {
-                // user selected a single directory
-
-                // Verify if valid Titanfall2 install location
-                let is_valid_titanfall2_install = await invoke("verify_install_location", { gamePath: selected }) as boolean;
-                if (is_valid_titanfall2_install) {
-                    this.$store.state.game_path = selected;
-                    // Check for Northstar install
-                    this.$store.commit('checkNorthstarUpdates');
-                }
-                else {
-                    // Not valid Titanfall2 install
-                    ElNotification({
-                        title: 'Wrong folder',
-                        message: "Selected folder is not a valid Titanfall2 install.",
-                        type: 'error',
-                        position: 'bottom-right'
-                    });
-                }
-            }
```

Comment:
Shouldn't the user still be able to update the path in the settings menu? ^^

Reply:
This code was actually moved in the `updateGamePath` method (https://github.com/GeckoEidechse/FlightCore/pull/7/files#diff-32bc04ae938110cf119ad14d3b3c167056516fea1f071eb1d45c1804dbc7ed4dR73), so it's used when:
* user clicks omnibutton when game folder was not found
* user clicks folder button in settings view